functions-methods.js

```diff
@@ -21,6 +28,26 @@
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
 
+const typeOfEmail = email => {
+    const studentEmail = email.includes("education");
+    const employeeEmail = email.includes("novi.nl");
+    const external = email.includes("outlook.com");
+    if (studentEmail === true){
```

Comment:
`if (studentEmail === true){` is hetzelfde als `if (studentEmail){` omdat `studentEmail` al een boolean is. Ik zou het laatste prefereren.